index.html

```diff
@@ -9,6 +9,14 @@
         <link rel="stylesheet" href="style.css">
     </head>
     <body>
+        <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}
```

Comment:
This looks good, can you put this style in style.css?
Then we are good to go :)